docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-metrics-ui.asciidoc

```diff
@@ -0,0 +1,68 @@
+[role="xpack"]
+[[ootb-ml-jobs-metrics-ui]]
+= Metrics {anomaly-detect} configurations
+++++
+<titleabbrev>Metrics</titleabbrev>
+++++
+
+// tag::metrics-jobs[]
+These {anomaly-jobs} can be created in the
+{kibana-ref}/xpack-infra.html[Metrics app] in {kib}. When you create one of 
```

Comment:
> {kibana-ref}/xpack-infra.html[Metrics app] in {kib}...

@EamonnTP I think I remember hearing that Observability pages like this were moving out of the Kibana guide.  Is there somewhere we should link to in the Observability guide instead?

Reply:
@szabosteve @lcawl That's correct. Ideally, once we have the anomaly detection topic within the "Analyze metrics" section of the Observability guide, we should link to that topic. Or for the mean time, you could link to the new "Analyze metrics" section - the [PR](https://github.com/elastic/observability-docs/pull/134) will get merged later this week. The link would be `{observability-guide}/analyze-metrics.html[Metrics app] in {kib}`.

Reply:
Created an [issue](https://github.com/elastic/observability-docs/issues/137) for this in the Observability docs repo.

Reply:
I changed to `{observability-guide}/analyze-metrics.html`. I'll modify it again in another PR once the Inspect metrics anomalies topic is published.